README.md

```diff
@@ -99,7 +99,7 @@ func main() {
 > ```shell
 > cd example && FINGERPRINT_API_KEY=SECRET_API_KEY VISITOR_ID=VISITOR_ID_EXAMPLE go run getVisits.go
 > ```
```

Comment:
Consider splitting [Installation & Usage](https://github.com/fingerprintjs/fingerprint-pro-server-api-go-sdk/blob/61d5a6a189d20a882acbecbcc8e20b07d39cc464/README.md#installation--usage) into "How to install" and "Getting started" just for consistency sake

Reply:
In GO, usually you just import the library and then it is installed automatically when you build the app, so I don't think that splitting this section here will make sense